deletiontests/azure/azure_delete_test.go

```diff
@@ -0,0 +1,188 @@
+package ingress
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/apimachinery/pkg/api/errors"
+	capzV1alpha3 "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
+	capiv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
```

Comment:
```suggestion
	capz "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
	capi "sigs.k8s.io/cluster-api/api/v1alpha3"
```
I'd propose that we use non-versioned import alias for these. I think Jose started this already earlier and I followed this with my e2e PR as well. It makes future dependency update easier when the actual code doesn't need to change but we just update imports.

Reply:
makes sense, done